Packs/McAfee-TIE/Integrations/McAfee-TIE/McAfee-TIE.py

```diff
@@ -176,6 +176,14 @@ def test():
         client.disconnect()
 
 
+def safe_get_file_reputation(tie_client, hash_param):
```

Comment:
the proper fix should be to return an `Unknown` score instead of raising an error

Reply:
Umm but what if McAfee will return some unknown error that we've not encountered yet?
Maybe add demisto log with the error and return unknown score?

Reply:
When indicator will not be found, the returned error from the API will be very specific. For example 400 error status or something like this.
You will have to check if it is that error, then return Unknown
Other exception, then fail the command